docs/ios.md

```diff
@@ -177,8 +177,8 @@ Set `OPENSSL_ROOT` to the install location:
 export OPENSSL_ROOT=/usr/local/ios/openssl
 ```
 
-Copy the static libraries (libcrypto.a and libssl.a) to the \$OPENSSL_ROOT/lib 
-and the header files to $OPENSSL_ROOT/include`.
+Copy the static libraries (libcrypto.a and libssl.a) to the `$OPENSSL_ROOT/lib` 
+and the header files to `$OPENSSL_ROOT/include`.
```

Comment:
This shouldn't be necessary, you should be able to do a `make install PREFIX=$OPENSSL_ROOT`?

Reply:
Yes, you're right. I found that by configuring for a native build then pointing to the cross-compiler SDK, it will build with out linker errors and will install with make install. I will update the doc. Thanks.